spec/rubocop/cop/rails/render_plain_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::RenderPlain do
+  subject(:cop) { described_class.new }
+
+  it 'registers an offense when using `render text:`' do
+    expect_offense(<<~RUBY)
+      render text: 'Ruby!'
+      ^^^^^^^^^^^^^^^^^^^^ Prefer `render plain:` over `render text:`.
+    RUBY
+
+    expect_no_corrections
```

Comment:
Why is it not auto-corrected?

Reply:
`render text: 'Ruby!'` - without explicit `content_type`, it sets MIME type to `text/html`
and autocorrecting it to `render plain: 'Ruby!'` sets MIME type to `text/plain`, which can be unexpected. 
So we can autocorrect only when explicit `content_type` is specified.

Link: https://rails.rubystyle.guide/#plain-text-rendering

Reply:
Thanks for your explanation. I think that this cop can provide an option like `MimeTypeCompatibility` (or `ContentTypeCompatibility`). The default suggests safety. e.g.:

```ruby
# @example
#   # bad - explicit MIME type to `text/plain`
#   render text: 'Ruby!', content_type: 'text/plain'
#
#   # good - short and precise
#   render plain: 'Ruby!'
#
#   # good - explicit MIME type not to `text/plain`
#   render text: 'Ruby!', content_type: 'text/html'
#
# @example MimeTypeCompatibility: true (default)`
#   # good - sets MIME type to `text/html`
#   render text: 'Ruby!'
#
# @example`MimeTypeCompatibility: false`
#   # bad - sets MIME type to `text/html`
#   render text: 'Ruby!'
```

As for the cop name, `PlainTextRendering` cop or `RenderPlainText` cop looks better according to the style guide rule name. (I'm sorry for changing from the previous proposal cop name 💦 )